### ISSUE TITLE
Add quick way to go to sub workflow

### ISSUE THREAD

**Issue by GuyShaanan** [nokia/CloudFlow#29]
Today when a task is actually a workflow you have to click it, scroll to bottom on the right panel, find the execution and click the icon to go to that execution.

Need to find a quick(er) way to achieve this (like on the task itself or somewhere else that doesn't require scrolling).

@tshtilman

**Comment by MosheElisha** [comment]
I agree but we need to keep in mind that there might be several WF executions in task.
For example, if the task has with-items or retry...

**Comment by tshtilman** [comment]
if there is only one we can definitely do it the quick way
more than one- the old way

**Comment by rakhmerov** [comment]
Agree with Tomer. I think it'd be a good shortcut for task with just one subworkflow executions.
